src/crystal/system/random.cr

```diff
@@ -14,7 +14,10 @@ end
   require "./unix/getrandom"
 {% elsif flag?(:openbsd) %}
   require "./unix/arc4random"
-{% else %}
-  # TODO: restrict on flag?(:unix) after crystal > 0.22.0 is released
+{% elsif flag?(:unix) %}
```

Comment:
Does any platforms actually set :unix as a flag? AFAICT this condition applies to FreeBSD and Darwin atm but in other places we do specifically check for those two OSs